src/router/index.js

```diff
@@ -143,7 +143,7 @@ router.beforeEach((to, from, next) => {
   } else if (to.name === "Play" && currentUser) {
     get(child(dbRef, `/users/${currentUser.uid}`)).then(snapshot => {
       const userData = snapshot.val();
-      if (!userData.taken_tutorial && config.needsTutorial) {
+      if (!userData.taken_tutorial && config.tutorial.needsTutorial) {
```

Comment:
just wondering if `taken_tutorial` is meant to be camelCase?

Reply:
i followed the previous naming, but it should be updated. it has to be updated in a few places so i'll address it in another PR.